packages/vitest/src/node/reporters/reported-tasks.ts

```diff
@@ -5,6 +5,7 @@ import type {
   Suite as RunnerTestSuite,
   TaskMeta,
 } from '@vitest/runner'
+import type { ImportDuration } from '@vitest/runner/types/tasks'
```

Comment:
`ImportDuration` needs to be exported from the `@vitest/runner` entry point manually, `types/tasks` doesn't exist at runtime

It should also be re-exported from `public/index.ts`

Reply:
done